java/arcs/android/storage/database/DatabaseImpl.kt

```diff
@@ -804,6 +908,27 @@ class DatabaseImpl(
         return typeMap
     }
 
+    @VisibleForTesting
```

Comment:
It might make sense to use `otherwise = VisibleForTesting.NONE` to 'hide' this outside of tests.

Reply:
Done